packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.constant.js

```diff
@@ -53,6 +53,15 @@ export const CHART = {
     tooltips: {
       mode: 'label',
       intersect: false,
+      callbacks: {
+        title: (context) => {
+          const timeZoneOffset =
+            (new Date(context[0].label).getTimezoneOffset() / 60) * -1;
```

Comment:
Just to be sure there won't be an issue, getTimezoneOffset is getting the timezone offset of the local zone and not the zone of the date:
For example: if the date is 2024-06-06T10:00:000+01:00 but the code is executed on the timezone UTC+2, the computed timezoneOffset will be 120 and not 60. 
Only the date is used which allow to get the good offset depending if you are in Daylight saving time or not.

Reply:
We need to display the date in the local zone of the user. For the moment, the date displayed is, for example, 2024-02-08T05:55:00Z which is not readable for users.